HACKING.md

```diff
@@ -161,6 +161,18 @@ To run all game tests locally, set `OPENENROTH_MM7_PATH` environment variable to
 
 Changing game logic might result in failures in game tests because they check random number generator state after each frame, and this will show as `Random state desynchronized when playing back trace` message in test logs. This is intentional – we don't want accidental game logic changes. If the change was actually intentional, then you might need to either retrace or re-record the traces for the failing tests. To retrace, run `OpenEnroth retrace <path-to-trace.json>`. Note that you can pass multiple trace paths to this command.
 
+Console Commands
+-----------------
+The game features a console capable of executing various Lua script commands. To activate this feature, please follow these steps:
+
+1. Copy the `resources/ui` folder to the game's installation directory.
+2. Launch the game.
+3. Begin by loading an existing game or creating a new one.
+4. Once in-game, press the `\` (backslash) key to open the DebugView.
```

Comment:
I think it's `~` not `\`?

Reply:
Whoops! you're right! 🤦I have a different keyboard setup 😅